services/QuillLMS/client/app/bundles/PremiumHub/components/usage_snapshots/snapshotCount.tsx

```diff
@@ -1,5 +1,7 @@
 import * as React from 'react'
 
+import md5 from 'md5'
```

Comment:
I believe that JS has a [native digest function](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest). Let's use that if possible.

Reply:
Per our discussion on Slack, the difficulty of getting native stuff to work both in-browser and in test environment, I'm sticking with a separate md5 library for now.